src/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixer.php

```diff
@@ -88,8 +88,6 @@ private function addRequiresCover(Tokens $tokens, $startIndex)
             : '';
 
         $prevIndex = $tokens->getPrevNonWhitespace($index);
-        $doc = null;
-        $docIndex = null;
```

Comment:
FYI:
i added it explicitly, i hate assigning a value in a `if` block to a variable that was not yet defined. `if` should create a variable-scope, but it doesn't in php :(

Reply:
When I see initialization like this I expect that there might be `if` condition which will override a value, but if not entering that condition will not and we will have to use the defaults. And in this case we always override it.